src/ontology/mondo-edit.obo

```diff
@@ -39331,17 +39331,13 @@ is_a: MONDO:0040677 {source="NCIT:C38760"} ! invasive carcinoma
 id: MONDO:0002497
 name: obsolete food allergy
 def: "OBSOLETE. Gastrointestinal disturbances, skin eruptions, or shock due to allergic reactions to allergens in food." [MESH:D005512]
-synonym: "allergic disease from food material" EXACT []
-synonym: "allergy of food material" EXACT [MONDO:patterns/allergy]
-synonym: "food hypersensitivity" EXACT [DOID:3044]
-synonym: "food material allergic disease" EXACT [MONDO:patterns/environmental_stimulus]
-xref: DOID:3044 {source="MONDO:obsoleteEquivalent"}
-xref: EFO:1001890 {source="MONDO:obsoleteEquivalent"}
+xref: DOID:3044
+xref: EFO:1001890
 xref: ICD9:995.3 {source="MONDO:relatedTo", source="MONDO:i2s"}
-xref: MESH:D005512 {source="DOID:3044", source="MONDO:obsoleteEquivalent"}
+xref: MESH:D005512 {source="DOID:3044"}
 xref: SCTID:157801005 {source="DOID:3044"}
 xref: SCTID:213018006 {source="DOID:3044"}
-xref: SCTID:414285001 {source="DOID:3044", source="MONDO:obsoleteEquivalent"}
+xref: SCTID:414285001 {source="DOID:3044"}
 xref: SCTID:52332009 {source="DOID:3044"}
 xref: UMLS:C0016470 {source="DOID:3044", source="MONDO:notFoundInDiseaseSubset"}
 property_value: IAO:0000231 "out of scope" xsd:string
```

Comment:
this should have a term replaced by that points to the new term you reinstated

Reply:
There is a "replaced by". I guess the question is: should I remove all the existing x-ref from the obsoleted term since they are on the "replaced by" term?

Reply:
ah, I see it!

We should have a new obs reason that is something like 'term reinstated'. Do you agree?

I removed the xrefs, per your question (because I opened up the file to look at the changes, so I figured I'd do the eidts since I already had the branch  open)

Reply:
@sabrinatoro do you think we should have a new obs reason that is something like 'term reinstated'? (But I worry we haven't done this consistently and it may be hard to find the other few terms that were reinstated, so maybe we don't want this?) If not, feel free to merge this PR

Reply:
@sabrinatoro ⬆️ 👀

Reply:
@nicolevasilevsky Generally, I think we should avoid as much as possible reinstating terms to avoid confusion. I think that adding another code would be confusing, and as you mentioned, difficult to keep consistent. I think we should make sure we have a GH issue referred to on both terms. (I don't know if it is the case here. I will check and merge or add.) 
Thank you!